src/it/add-third-party-no-encoding/invoker.properties

```diff
@@ -19,5 +19,5 @@
 # <http://www.gnu.org/licenses/lgpl-3.0.html>.
 # #L%
 ###
-invoker.goals=clean license:add-third-party
+invoker.goals=clean org.codehaus.mojo:license-maven-plugin:@pom.version@:add-third-party
```

Comment:
I think you want `${project.groupId}:${project.artifactId}:${project.version}:add-third-party` see https://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#invokerPropertiesFile